openduck-py/openduck_py/settings.py

```diff
@@ -0,0 +1 @@
+IS_DEV = True
```

Comment:
wasn't this added elsewhere? thought I remembered a PR from Will.

Reply:
ah yeah https://github.com/uberduck-ai/openduck/blob/main/openduck-py/openduck_py/settings/__init__.py